HUD.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include <SFML/Graphics.hpp>
+
+
+class HUD

```

Comment:
As far as I can see, this class is not used? What happened?

Reply:
The game is not actually finished: I still have to implement the HUD and the TimerBar, but wanted to get your feedback before I proceed